### PR TITLE
Validate PHP version mismatch

### PR DESCRIPTION
**Pull request by murrant**
If the webserver does not match the cli version this means they recently upgraded and have not restarted the service.

This can cause errors with some cached data.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: LibreNMS/Validations/Php.php [hunk 1]
```diff
@@ -56,6 +56,12 @@ private function checkVersion(Validator$validator)
         if (Config::get('update') && version_compare(PHP_VERSION, self::PHP_MIN_VERSION, '<')) {
             $validator->warn("PHP version " . self::PHP_MIN_VERSION . " is the minimum supported version as of " . self::PHP_MIN_VERSION_DATE . ". We recommend you update PHP to a supported version (" . self::PHP_RECOMMENDED_VERSION . " suggested) to continue to receive updates. If you do not update PHP, LibreNMS will continue to function but stop receiving bug fixes and updates.");
         }
+
+        $web_version = PHP_VERSION;
+        $cli_version = rtrim(shell_exec('php -r "echo PHP_VERSION;"'));
+        if (version_compare($web_version, $cli_version, '!=')) {
+            $validator->fail("PHP version of your webserver ($web_version) does not match the cli version ($cli_version)", "If you updated PHP recently, restart php-fpm or apache to switch to the new version");
```
Comment (SourceDoctor) [inline LibreNMS/Validations/Php.php:63]: mmh ... "restart php-fpm, apache. nginx or what webserver you use..  " ?

Comment (murrant) [reply LibreNMS/Validations/Php.php:63]: restarting nginx will never have any affect on the php version.

Comment (murrant) [reply LibreNMS/Validations/Php.php:63]: and you can use php-fpm with apache

Comment (Jellyfrog) [reply LibreNMS/Validations/Php.php:63]: To nitpick you can run php other ways than php-fpm in that case, Nginx unit is one way :)  
Point is we can't cover all use cases, some things need to be left to the sysadmin.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-64-release-changelog-may-2020/12290/1

**Pull request merged**